interface/xcmp_xpr.py

```diff
@@ -0,0 +1,246 @@
+import threading
+from binascii import hexlify
+from radioState import RadioState
+from logger import Logger
+import time
+import queue
+from trboxcmp.xcmp import Xcmp
```

Comment:
We'll need to coordinate getting this library into the repo, or some other way to distribute it.

Reply:
trboxcmp is on pypi: https://pypi.org/project/trboxcmp/

Reply:
Will add to list of dependencies, thanks!